docs/intro.rst

```diff
@@ -15,3 +19,166 @@ verifiability, performance, and breadth of adoption. Imath is not
 intended to be a comprehensive linear algebra or numerical analysis
 package.
 
+Imath is not `Eigen <https://eigen.tuxfamily.org>`_! It's not a
+full-featured linear algebra package, and it doesn't represent vectors
+and matrices of arbitrary dimension. Its greatest utility is as a
+geometric data representation, primarily for 2D images and 3D scenes
+and coordinate transformations, along with an accompanying set of
+utility methods and functions.
+
+Quick Links:
+------------
+
+- Download: https://github.com/AcademySoftwareFoundation/Imath
+- License: `BSD License <https://github.com/OpenImageIO/oiio/blob/master/LICENSE.md>`_
+
+
+History
+-------
+
+Imath originated at Industrial Light & Magic in the late 1990's and
+early 2000's, and it was originally distributed publicly as a
+component of
+`OpenEXR <https:://github.com/AcademySoftwareFoundation/openexr>`_.
+
+Imath is Version 3 because it was previously distributed as a
+component of OpenEXR v1 and v2.
+
+Example
+-------
+
+A basic program:
+
+.. code-block::
+
+   #include <Imath/ImathVec.h>
+   #include <iostream>
+   
+   int
+   main()
+   {
+       Imath::V3f v (3.0, 4.0, 5.0);
+   
+       v.normalize();
+
+       Imath::M33f M;
+       M.translate (1.0, 2.0, 3.0);
+       Imath::V3f p = v * M;
+
+       std::cout << "What's your vector, Victor? " << p << std::endl;
+
+       return 0;
+   }
+
+Matrices Are Row-Major
+----------------------
+
+Imath stores matrices in row-major layout, originally inspired by
+compatibility with OpenGL matrices. A matrix representing a
+homogeneous transform has 0's in the right-name column and the
```

Comment:
Right-"hand". The w component of the right hand column has a 1 in it, though. Maybe just saying "a right hand column of [0,0,0,1] is sufficient

Reply:
Fixed.